portal/static/portal/sass/modules/_colours.scss

```diff
@@ -2,6 +2,7 @@
 $color-teacher-blue: #0c52a3;
 $color-student-orange: #f37300;
 $color-text-grey: #4a4a4a;
+$color-aimmo-materials: #6c6a6c;
```

Comment:
Is it necessary to introduce another color here? if so can we make it give it a more generic name?

Reply:
I gave it a specific name because it's temporary, and James told me to use this one. I'm open minded to using $color-text-grey but I don't know what @j4mesholland would think?

Reply:
I don't think there is such a thing as a temporary colour :P

Reply:
removing that colour and now using  $color-text-grey: #4a4a4a approved by @j4mesholland